### PR TITLE
style: reformat to standardise code

### PR DESCRIPTION
**Pull request by AndrewBB-BBD**

### CODE REVIEW

**Review by ms-si-Singh — APPROVED** [review]

File: SCA_DB_script.sql [hunk 1]
```diff
@@ -50,33 +50,33 @@ CREATE TABLE dbo.[Assailant] (
...
-  [Location_ID] INT NOT NULL,
-  [Date] DATE NOT NULL,
-  [Time] TIME NOT NULL,
-  [Description] varchar(150) NOT NULL,
+  [Incident_ID] [INT] IDENTITY (1,1) NOT NULL PRIMARY KEY,
+  [User_ID] [INT] NOT NULL,
+  [Location_ID] [INT] NOT NULL,
+  [Date] [DATE] NOT NULL,
+  [Time] [TIME] NOT NULL,
+  [Description] varchar(150) NULL,
```
Comment (nikitasmal) [inline SCA_DB_script.sql:79]: Should this varchar not also be in square brackets like the rest?

**Review by AmyPegramBBD — APPROVED** [review]

File: SCA_DB_script.sql (hunk 1 above)
Comment (AndrewBB-BBD) [reply SCA_DB_script.sql:79]: It should! Thank you :)

**Review by nikitasmal — APPROVED** [review]
Besides the varchar on line 79 that is not is square brackets everything looks good.

File: SCA_DB_script.sql (hunk 1 above)
Comment (nikitasmal) [reply SCA_DB_script.sql]: Pleasure, sorry didn't see this reply. Also sent a message on teams.

**Pull request merged**